build.gradle

```diff
@@ -87,9 +87,12 @@ dependencies {
     implementation project(':alert-common')
     implementation project(':alert-database')
     implementation project(':channel')
+    implementation project(':component')
     implementation project(':provider')
+    implementation project(':workflow')
     implementation project(':web')
 
+
     // Spring
```

Comment:
We we able to remove some of the dependencies we copied over to the `web` sub-project?

Reply:
Went through all of the build.gradle files and updated the configurations for all the libraries.